universe/interface.go

```diff
@@ -393,6 +393,11 @@ type Proof struct {
 	MultiverseInclusionProof *mssmt.Proof
 }
 
+// LowerBoundByteSize returns the lower bound on the byte size of the proof.
+func (p *Proof) LowerBoundByteSize() uint64 {
```

Comment:
could also add some test coverage  for the `LowerBoundByteSize` function of `universe.Proof`

Reply:
`universe.Proof.LowerBoundByteSize` just delegates to `fn.LowerBoundByteSize`, and the underlying function already has test coverage.

Maybe I've misunderstood what you mean.

Reply:
I mean we should create dummy `universe.Proof` instances and verify that an iteration of `fn.LowerBoundByteSize` over it actually returns the size of it.

I guess if we're very confident on the `fn` function coverage this is actually not needed. Would only be a sanity check that it can iterate a proof properly.